packages/code-studio/vite.config.ts

```diff
@@ -54,10 +54,8 @@ export default defineConfig(({ mode }) => {
   // It is possible to add one with a custom middleware though if this list grows
   if (env.VITE_PROXY_URL) {
     [
-      env.VITE_CORE_API_URL,
-      env.VITE_NOTEBOOKS_URL,
-      env.VITE_LAYOUTS_URL,
-      env.VITE_MODULE_PLUGINS_URL,
+      path.resolve(env.BASE_URL, env.VITE_CORE_API_URL),
+      path.resolve(env.BASE_URL, env.VITE_MODULE_PLUGINS_URL),
```

Comment:
Should any of the above proxy elements include the `BASE_URL`? I guess styleguide is fine without it. Are the websocket requests always to the server root?

Reply:
We always connect to the server root for GRPC.
I did break the styleguide though.